motion_computation/config/parameters.yaml

```diff
@@ -22,11 +22,11 @@ enable_bsm_processing: false
 
 # Boolean: If true then PSM messages will be converted to ExternalObjects. 
 #          If other object sources are enabled, they will be synchronized but no fusion will occur (objects may be duplicated)
-enable_psm_processing: false
+enable_psm_processing: true
 
 # Boolean: If true then MobilityPath messages will be converted to ExternalObjects. 
 #          If other object sources are enabled, they will be synchronized but no fusion will occur (objects may be duplicated)
-enable_mobility_path_processing: true
+enable_mobility_path_processing: false
```

Comment:
We should leave this on true, at the moment. For actually testing only CP we can set it to false in the vehicle.